### ISSUE TITLE
Can't use IEnumerable, multi fields in an URL-encoded body

### ISSUE THREAD

**Issue by Sergio0694** [reactiveui/refit#621]
I noticed that Refit doesn't handle `IEnumerable<T>` values in classes (possibly related to #587 ) and that it's pretty much impossible to use arrays/lists in URL-encoded body contents.
I'm trying to write [this Imgur API](https://apidocs.imgur.com/#8f89bd41-28a1-4624-9393-95e12cec509a) with Refit, but no matter what I tried it just wouldn't work (this does work when trying with other programs to test REST APIs though).

What's causing trouble is that `deletehashes[]` field in the URL-encoded body, which needs to be duplicated for each additional field. The issue is that:
1) If I place a `string[]` parameter in the class to use as the URL-encoded body, I stumble upon the issue in #587 and the serialization doesn't properly handle the value of that property.
2) I can't just use a `Dictionary<string, object>` as the body, as I can have multiple values that share the `deletehashes[]` key, so they obviously can't be in a dictionary.

Basically, right now I can't see how it would be possible to get this working with Refit.
Ideally, I'd have a `string[]` property in my class, passed as the URL-encoded body, like this:
```csharp
public sealed class AlbumRequest
{
    [AliasAs("deletehashes[]")]
    [Query(CollectionFormat.Multi)]
    public string[] ModHashes { get; set; }
}
```
And then if I assigned, say, `new[] { "First", "Second", "Third" }` to it, the resulting form encoded body would contain:

> modhashes[] = First
> modhashes[] = Second
> modhashes[] = Third

Is there any way to achieve this in Refit, or would it be possible to add support for this?
Thanks!

**Comment by Sergio0694** [comment]
Nevermind, turns out this has been fixed already in a very recent NuGet update, and I was using version `4.6.58`, which didn't have the fix yet 😅
Keep up the good work!

**Comment by ahmedalejo** [comment]
Hi @Sergio0694  what version did find with this resolved?

I file in a similar issue a while ago with no feedback
#587

**Comment by Sergio0694** [comment]
Hi @ahmedalejo - I've updated to package `4.6.90`, try it out and hopefully it should solve your issue as well! 👍
